app/assets/javascripts/views/file_uploader_manager.js.coffee

```diff
@@ -0,0 +1,12 @@
+class ELMO.Views.FileUploaderManager extends ELMO.Views.ApplicationView
```

Comment:
Is this class needed?  It looks like the only use of it is as the `listener` in the uploader view.  Can we just log these messages from the uploader view?  I'm also not sure if that `events` hash does anything here.

Reply:
removed; and renamed old media uploader manager to file uploader manager